src/components/protection/AddProtectionSingle.vue

```diff
@@ -494,6 +494,7 @@ export default class AddProtectionSingle extends BaseComponent {
     this.loading = true;
     try {
       await Promise.all([
+        new Promise(r => setTimeout(r, 1000)),
```

Comment:
What's going on here?

Reply:
Ah okay, this forces the UI to wait at least a second before it resolves then loading is set to false, will replace this with just ```wait```